### ISSUE TITLE
Exception on onPacketSending(PacketEvent) for React

### ISSUE THREAD

**Issue by xenos5077** [VolmitSoftware/React#191]
```
[10:31:06] [Server thread/ERROR]: [React] Unhandled exception occured in onPacketSending(PacketEvent) for React
com.comphenix.protocol.reflect.FieldAccessException: No field with type long exists in class PacketPlayOutKeepAlive.
	at com.comphenix.protocol.reflect.StructureModifier.readInternal(StructureModifier.java:219) ~[ProtocolLib.jar:4.3.0]
	at com.comphenix.protocol.reflect.StructureModifier.read(StructureModifier.java:198) ~[ProtocolLib.jar:4.3.0]
	at react.protocol.ProtocolAdapter$2.onPacketSending(ProtocolAdapter.java:92) ~[React-6.0.1.jar:?]
	at com.comphenix.protocol.injector.SortedPacketListenerList.invokeSendingListener(SortedPacketListenerList.java:195) [ProtocolLib.jar:4.3.0]
	at com.comphenix.protocol.injector.SortedPacketListenerList.invokePacketSending(SortedPacketListenerList.java:149) [ProtocolLib.jar:4.3.0]
	at com.comphenix.protocol.injector.PacketFilterManager.handlePacket(PacketFilterManager.java:588) [ProtocolLib.jar:4.3.0]
	at com.comphenix.protocol.injector.PacketFilterManager.invokePacketSending(PacketFilterManager.java:564) [ProtocolLib.jar:4.3.0]
	at com.comphenix.protocol.injector.netty.ProtocolInjector.packetQueued(ProtocolInjector.java:338) [ProtocolLib.jar:4.3.0]
	at com.comphenix.protocol.injector.netty.ProtocolInjector.onPacketSending(ProtocolInjector.java:298) [ProtocolLib.jar:4.3.0]
	at com.comphenix.protocol.injector.netty.ChannelInjector.processSending(ChannelInjector.java:386) [ProtocolLib.jar:4.3.0]
	at com.comphenix.protocol.injector.netty.ChannelInjector.access$500(ChannelInjector.java:77) [ProtocolLib.jar:4.3.0]
	at com.comphenix.protocol.injector.netty.ChannelInjector$5.handleScheduled(ChannelInjector.java:352) [ProtocolLib.jar:4.3.0]
	at com.comphenix.protocol.injector.netty.ChannelInjector$5.onMessageScheduled(ChannelInjector.java:320) [ProtocolLib.jar:4.3.0]
	at com.comphenix.protocol.injector.netty.ChannelProxy$2.schedulingRunnable(ChannelProxy.java:127) [ProtocolLib.jar:4.3.0]
	at com.comphenix.protocol.injector.netty.EventLoopProxy.execute(EventLoopProxy.java:95) [ProtocolLib.jar:4.3.0]
	at net.minecraft.server.v1_12_R1.NetworkManager.a(NetworkManager.java:194) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
	at net.minecraft.server.v1_12_R1.NetworkManager.sendPacket(NetworkManager.java:143) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
	at net.minecraft.server.v1_12_R1.PlayerConnection.sendPacket(PlayerConnection.java:1131) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
	at net.minecraft.server.v1_12_R1.PlayerConnection.e(PlayerConnection.java:182) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
	at net.minecraft.server.v1_12_R1.NetworkManager.a(NetworkManager.java:233) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
	at net.minecraft.server.v1_12_R1.ServerConnection.c(ServerConnection.java:140) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:842) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:405) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:678) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_161]
[10:31:06] [Server thread/ERROR]: Parameters: 
  net.minecraft.server.v1_12_R1.PacketPlayOutKeepAlive@7c750d64[
    a=247700
  ]
```

**Comment by cyberpwnn** [comment]
For right now, i would suggest turning off packet sampling in the config.yml then reloading. But i will fix it.

**Comment by cyberpwnn** [comment]
Fixed in 6.0.2
